examples/python/auth/helloworld.proto

```diff
@@ -0,0 +1,38 @@
+// Copyright 2015 gRPC authors.
```

Comment:
Optional: It looks like the `examples` directory isn't included in google3. Maybe we could use a symlink instead?

Reply:
Made it a symlink. I worried about how will our CI react. Let's see if tests are okay.

Reply:
As I recall, Piper was the only thing that choked on a symlink the last time we tried it. If CI fails, feel free to change it back. But if it works, I definitely think it's worth reducing the duplication.